ex1.py

```diff
@@ -0,0 +1,18 @@
+
+try:
+    def calculate_average ():
+        counter = 0
+        sum = 0
+        function_stop = "q"
+        num = input("enter num if you want to stop enter " + function_stop + " ")
```

Comment:
use string formating instead!
f"enter num if you want to stop enter  {NAME_OF_VAR}   "